eu.betterbird.Betterbird.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
has this been submitted for inclusion upstream?

Reply:
Yes: https://github.com/Betterbird/thunderbird-patches/pull/69

Will be removed here once available upstream.